ormconfig.ts

```diff
@@ -12,7 +12,7 @@ const config: PostgresConnectionOptions = {
   password,
   database,
   entities: ['dist/src/**/*.entity.js'],
-  synchronize: !isProduction,
```

Comment:
did you mean to commit this? I know this is annoying when trying to test migrations but I'm not sure about the draw backs in terms of normal development?

Reply:
Yeah I mean if we always default run migrations on start up then that probably works. It is also oddly taking the environment from Rollbar env.... It was a weird config to synchronise on staging and not on prod.

Reply:
Yeah exactly. Cool will make the change.